### DISCUSSION TITLE
Poe窗口挡住了，需要更新一下。

### DISCUSSION THREAD

**Discussion by enlivatech** [tw93/Pake#459] [General]
![CleanShot 2023-06-07 at 08 25 37@2x](https://github.com/tw93/Pake/assets/20806624/ace572d4-8d32-431f-bfda-c7209aba9f0d)

**Comment by tw93** [comment]
谢谢提醒，最近版本我更新一下

**Comment by tw93** [comment]
已经更新，去下载最新版本就好 https://github.com/tw93/Pake/releases/latest/download/Poe.dmg
